src/snowcli/cli/common/flags.py

```diff
@@ -14,6 +15,23 @@
 _CONNECTION_SECTION = "Connection configuration"
 _CLI_BEHAVIOUR = "Global configuration"
 
+ProjectDefinitionOption = typer.Option(
+    None,
```

Comment:
```suggestion
    None,
```
Can it be `path.cwd()`?